linode/accountavailabilities/datasource_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build integration
+//go:build (integration || accountavailabilities) && !optional && !long_running
```

Comment:
Just to make sure I'm understanding this fully, is the intention here to disable this test if long-running tests or optional tests are enabled?

Reply:
Yeah exactly, and I wanted to give an option to run each package separately if we needed to. 
E.g. 
- `accountavailabilities` will run just tests in the package
- `integration` will run all tests excluding tests marked with long_running and optional
- specifying either optional or long_running will run exclude everything except that is marked those tags

Reply:
I changed the logic a bit for better ease of use so scratch everything I said above. Updated the description